src/desktopvirtualization/azext_desktopvirtualization/generated/commands.py

```diff
@@ -9,47 +9,61 @@
+    cf_workspace,
+    cf_application_group,
+    cf_host_pool,
+)
 
 
-def load_command_table(self, _):
+desktopvirtualization_application_group = CliCommandType(
+    operations_tmpl='azext_desktopvirtualization.vendored_sdks.desktopvirtualization.operations._application_groups_operations#ApplicationGroupsOperations.{}',
+    client_factory=cf_application_group,
+)
+
+
+desktopvirtualization_host_pool = CliCommandType(
+    operations_tmpl='azext_desktopvirtualization.vendored_sdks.desktopvirtualization.operations._host_pools_operations#HostPoolsOperations.{}',
+    client_factory=cf_host_pool,
+)
+
+
+desktopvirtualization_workspace = CliCommandType(
+    operations_tmpl='azext_desktopvirtualization.vendored_sdks.desktopvirtualization.operations._workspaces_operations#WorkspacesOperations.{}',
+    client_factory=cf_workspace,
+)
 
-    from azext_desktopvirtualization.generated._client_factory import cf_workspace
-    desktopvirtualization_workspace = CliCommandType(
-        operations_tmpl='azext_desktopvirtualization.vendored_sdks.desktopvirtualization.operations._workspace_operatio'
-        'ns#WorkspaceOperations.{}',
-        client_factory=cf_workspace)
-    with self.command_group('desktopvirtualization workspace', desktopvirtualization_workspace,
-                            client_factory=cf_workspace, is_experimental=True) as g:
-        g.custom_command('list', 'desktopvirtualization_workspace_list')
-        g.custom_show_command('show', 'desktopvirtualization_workspace_show')
-        g.custom_command('create', 'desktopvirtualization_workspace_create')
-        g.custom_command('update', 'desktopvirtualization_workspace_update')
-        g.custom_command('delete', 'desktopvirtualization_workspace_delete')
 
-    from azext_desktopvirtualization.generated._client_factory import cf_application_group
-    desktopvirtualization_application_group = CliCommandType(
-        operations_tmpl='azext_desktopvirtualization.vendored_sdks.desktopvirtualization.operations._application_group_'
-        'operations#ApplicationGroupOperations.{}',
-        client_factory=cf_application_group)
+def load_command_table(self, _):
+
     with self.command_group('desktopvirtualization applicationgroup', desktopvirtualization_application_group,
-                            client_factory=cf_application_group, is_experimental=True) as g:
+                            client_factory=cf_application_group) as g:
         g.custom_command('list', 'desktopvirtualization_applicationgroup_list')
         g.custom_show_command('show', 'desktopvirtualization_applicationgroup_show')
         g.custom_command('create', 'desktopvirtualization_applicationgroup_create')
         g.custom_command('update', 'desktopvirtualization_applicationgroup_update')
-        g.custom_command('delete', 'desktopvirtualization_applicationgroup_delete')
+        g.custom_command('delete', 'desktopvirtualization_applicationgroup_delete', confirmation=True)
```

Comment:
Adding `confirmation=True` may lead to breaking change. Is this in line with expectations?

Reply:
`confirmation=True` is added by codegen, will remove them.